studies/templates/studies/study_responses.html

```diff
@@ -40,85 +40,180 @@
     <div class="row">
         <div class="col">{% include "studies/_response_nav_tabs.html" with active="individual" %}</div>
     </div>
+    <div class="row my-4">
+        <div class="col">
+            <p>
+                Data about
+                {{ summary_statistics.responses.accepted }}
+                {{ summary_statistics.responses.accepted|pluralize:"response is,responses are" }}
+                available.
+                {{ summary_statistics.responses.pending }}
+                additional
+                {{ summary_statistics.responses.pending|pluralize:"response,responses" }}
+                awaiting consent judgement;
+                {{ summary_statistics.responses.rejected }}
+                {{ summary_statistics.responses.rejected|pluralize:"consent,consents" }}
+                rejected.
+            </p>
+            <p>
+                <a href="{% url 'exp:study-responses-consent-manager' pk=study.pk %}">Goto consent manager</a>
+            </p>
+        </div>
+        <div class="col">
+            <div class="card bg-light">
+                <div class="card-body">
+                    <div>Selected response SHORTID (1/31/2025 2:24 PM).</div>
+                    <div>
+                        Parent/guardian feedback: had trouble hearing. I wrote a
+                        really long message but it's only to be allowed to go...
+                    </div>
+                    <div>Send a message to this family (EHJEC-kaitlin)</div>
```

Comment:
Should these divs have this static text about the selected response? Or should it be empty in the template?

Reply:
The text is a placeholder until we implement the correct functionality.